apps/website/get-started/developing/README.md

```diff
@@ -95,26 +90,22 @@ demos will be added in the near future.
 
 ### Angular
 
-To use Clarity Core with Angular follow the package [installation instructions](#installation). To work fully with the Angular compiler, we have also created an Angular package specifically to help the Angular compiler understand the custom element bindings. Normally custom elements require you to use the [`CUSTOM_ELEMENTS_SCHEMA`](https://angular.io/api/core/CUSTOM_ELEMENTS_SCHEMA) which allows non-Angular elements to be processed, but this has the side effect of not supporting strict checks in templates. Our `@cds/angular` package provides Angular component definitions so that you can use Clarity Core components like normal Angular components.
+To use Clarity Core with Angular follow the package [installation instructions](#installation).
 
-```bash
-npm install @cds/angular --save
-```
-
-Then you'll want to add the `CdsModule` to your AppModule, as well as import the component(s) you plan to use.
+Once installed add <code>CUSTOM_ELEMENTS_SCHEMA</code> to your application
```

Comment:
These instructions need to be reverted, as we have the @cds/angular package that doesn't require custom_elements_schema.